### DISCUSSION TITLE
An error occurred signing in: invalid input: invalid passcode.

### DISCUSSION THREAD

**Discussion by kimmy-wang** [artifacthub/hub#2718] [General]
An error occurred signing in: invalid input: invalid passcode. 

![截屏2023-01-30 17 05 58](https://user-images.githubusercontent.com/19725091/215434053-eb6e80ed-6cca-426b-a863-c5465b52ede0.png)

**Comment by tegioz** [comment]
Hi @kw214 👋 

That error indicates that the two-factor authentication code provided is not valid (it could be a code from a device or a recovery code). Could you provide a bit more context so that we can try to help you?

**Reply by kimmy-wang** [reply]
My guess is also that the two-factor authentication code is invalid, how can I turn off the two-factor authentication for my account.

**Reply by tegioz** [reply]
You can use your recovery codes to sign in and turn off two-factor authentication.

**Reply by kimmy-wang** [reply]
I lost my recovery codes.

**Reply by tegioz** [reply]
Unfortunately I don't think we can help recovering the account in that case. Our only intention is to protect users, hope you can understand.

Did you have any repositories listed from that account? If so, you can [claim their ownership](https://artifacthub.io/docs/topics/repositories/#ownership-claim) from a new account. If you encounter any problems during the process please let us know.
